### PR TITLE
[risk=low] Updating Test,Stable,Staging Cdr Version

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Cloned all the survey branching logic needed data generation changes and created data with the changes.

### CODE REVIEW

File: api/db-cdr/changelog-schema/db.changelog-25.xml [hunk 1]
```diff
@@ -17,10 +17,16 @@
...
         <constraints nullable="false"/>
       </column>
-      <column name="is_main" type="BOOLEAN">
+      <column name="question_order_number" type="INTEGER">
+        <constraints nullable="false"/>
+      </column>
+      <column name="path" type="VARCHAR(1000)">
+        <constraints nullable="false"/>
+      </column>
+      <column name="sub" type="INTEGER">
```
Comment (freemabd1) [inline api/db-cdr/changelog-schema/db.changelog-25.xml]: Liquibase should be used to incrementally change tables using different change sets. We should not be changing existing files every time the schema changes. These columns should be added in another changelog file.  Example: db.changelog-26.xml
```
<changeSet author="srushtigangireddy" id="changelog-26">
   <dropColumn tableName="survey_question_map" columnName="is_main"/>
    <addColumn tableName="survey_question_map">
      <column name="survey_order_number" type="INTEGER"/>
      <column name="question_order_number" type="INTEGER"/>
      <column name="sub" type="INTEGER"/>
    </addColumn>
    <modifyDataType tableName="survey_question_map" columnName="path" newDataType="VARCHAR(1000)"/>
</changeSet>
```

Comment (SrushtiGangireddy) [reply api/db-cdr/changelog-schema/db.changelog-25.xml]: Addressed thanks.

File: api/db-cdr/changelog-data/db.changelog-26.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  <changeSet author="srushtigangireddy" id="changelog-26">
+    <dropColumn tableName="survey_question_map" columnName="is_main"/>
+    <addColumn tableName="survey_question_map">
+      <column name="survey_order_number" type="INTEGER"/>
+      <column name="question_order_number" type="INTEGER"/>
+      <column name="sub" type="INTEGER"/>
+    </addColumn>
+    <modifyDataType tableName="survey_question_map" columnName="path" newDataType="VARCHAR(1000)"/>
+  </changeSet>
+</databaseChangeLog>
```
Comment (freemabd1) [inline api/db-cdr/changelog-data/db.changelog-26.xml]: Also, need an entry in the db-cdr/changelog-schema/db.changelog-master.xml for this file(db.changelog-26.xml)

**Comment by calbach** [comment]
@SrushtiGangireddy @freemabd should this be dropped?

**Comment by SrushtiGangireddy** [comment]
Yes, I am closing out this pr. Not needed anymore. Thanks.

**Pull request closed without merging**